### ISSUE TITLE
Unstable test: Submodule_RecognizedWithCorrectVersion

### ISSUE THREAD

**Issue by AArnott** [dotnet/Nerdbank.GitVersioning#125]
The `VersionOracleTests.Submodule_RecognizedWithCorrectVersion` test frequently fails with [this output](https://ci.appveyor.com/project/AArnott/nerdbank-gitversioning/build/2.0.4-beta+g4a5336233c#L719):

```
    VersionOracleTests.Submodule_RecognizedWithCorrectVersion [FAIL]
      System.ComponentModel.Win32Exception : The system cannot find the file specified
      Stack Trace:
           at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
           at System.Diagnostics.Process.Start()
           at SevenZipNET.WProcess.Execute()
           at SevenZipNET.SevenZipExtractor.ExtractAll(String destination, Boolean overwrite, Boolean keepStructure)
        src\NerdBank.GitVersioning.Tests\TestUtilities.cs(70,0): at Nerdbank.GitVersioning.Tests.TestUtilities.ExtractRepoArchive(String repoArchiveName)
        src\NerdBank.GitVersioning.Tests\VersionOracleTests.cs(18,0): at VersionOracleTests.Submodule_RecognizedWithCorrectVersion()
```

It causes PR builds to fail frequently.

**Comment by onovotny** [comment]
Seems like this can be closed now with your recent commit.

**Comment by AArnott** [comment]
Closed by e1be2046c9680029c68b974c357e635e847a5dd8

**Comment by AArnott** [comment]
It's still happening.
